kubernetes/updater/README.md

```diff
@@ -0,0 +1,23 @@
+## Teleport Kubernetes Updater (`teleport-kube-updater`)
```

Comment:
Can we move this and the teleport operator into the `/operator` folder?

My reasoning is that `/kubernetes` is a weird folder name because we have a lot of Kubernetes code that lies in `lib/kube`.

Can we use:
- `/operator/teleport`: Teleport operator
- `/operator/teleport-updater`: Teleport updater

With this structure it is easier to understand we have two operators

Reply:
Adding agent gives the following names:
- `teleport-kube-agent-updater`
- `kube-agent-updater`

It's consistent with the chart name `teleport-kube-agent` but might also be confusing in the same fashion: it kinda implies this only works with agents running kube-access service and not app-access, db-access, ...

Reply:
> It's consistent with the chart name teleport-kube-agent but might also be confusing in the same fashion: it kinda implies this only works with agents running kube-access service and not app-access, db-access, ...

It's true but I think it's clearer since it's expected to work only with our helm chart.

Reply:
As discussed on slack, I moved the updater to `/integrations/kube-agent-updater`, and renamed the executable `teleport-kube-agent-updater`. I will move the operator in `/integrations/operator` in a subsequent PR.